networks/suzuka/suzuka-full-node/src/partial.rs

```diff
@@ -54,180 +49,145 @@ impl <T : SuzukaExecutor + Send + Sync + Clone>SuzukaPartialNode<T> {
-
-            // get the transactions
-            let mut block_transactions = Vec::new();
-            for transaction in block.transactions {
-                let signed_transaction : SignedTransaction = serde_json::from_slice(&transaction.0)?;
-                let signature_verified_transaction = SignatureVerifiedTransaction::Valid(
-                    Transaction::UserTransaction(
-                        signed_transaction
-                    )
-                );
-                block_transactions.push(signature_verified_transaction);
-            }
-
-            // form the executable transactions vec
-            let block = ExecutableTransactions::Unsharded(
-                block_transactions
-            );
-            
-            // hash the block bytes
-            let mut hasher = sha2::Sha256::new();
-            hasher.update(&block_bytes);
-            let slice = hasher.finalize();
-            let block_hash = HashValue::from_slice(slice.as_slice())?;
-
-            // form the executable block and execute it
-            let executable_block = ExecutableBlock::new(
-                block_hash,
-                block
-            );
-            let block_id = executable_block.block_id;
-            self.executor.execute_block(
-                FinalityMode::Opt,
-                executable_block
-            ).await?;
-
-            println!("Executed block: {:?}", block_id);
-
-        }
-
-        Ok(())
-
-    }
-
-}
-
-impl <T : SuzukaExecutor + Send + Sync + Clone>SuzukaFullNode for SuzukaPartialNode<T> {
-    
-        /// Runs the services until crash or shutdown.
-        async fn run_services(&self) -> Result<(), anyhow::Error> {
-
-            self.executor.run_service().await?;
-
-            Ok(())
-
-        }
-
-        /// Runs the background tasks until crash or shutdown.
-        async fn run_background_tasks(&self) -> Result<(), anyhow::Error> {
+	pub async fn tick_write_transactions_to_da(&self) -> Result<(), anyhow::Error> {
+		// limit the total time batching transactions
+		let start_time = std::time::Instant::now();
+		let end_time = start_time + std::time::Duration::from_millis(100);
+
+		let mut transactions = Vec::new();
+
+		while let Ok(transaction_result) =
+			tokio::time::timeout(Duration::from_millis(100), self.transaction_receiver.recv()).await
+		{
+			match transaction_result {
+				Ok(transaction) => {
+					println!("Got transaction: {:?}", transaction);
+					let serialized_transaction = serde_json::to_vec(&transaction)?;
+					transactions.push(BlobWrite { data: serialized_transaction });
+				},
+				Err(_) => {
+					break;
+				},
+			}
+
+			if std::time::Instant::now() > end_time {
+				break;
+			}
+		}
+
+		if transactions.len() > 0 {
+			let client_ptr = self.light_node_client.clone();
+			let mut light_node_client = client_ptr.write().await;
+			light_node_client.batch_write(BatchWriteRequest { blobs: transactions }).await?;
+			println!("Wrote transactions to DA");
```

Comment:
Swap println from `log::info!`

Reply:
Will use  `tracing::trace!` calls, as we do not want this level of granularity at the info level.